packages/ai-sandbox/tests/with-sandbox-hooks.test.ts

```
@@ -3,13 +3,15 @@ import { provideSandboxRuntime } from '@tanstack/ai/adapter-internals'
 import { resolveDebugOption } from '@tanstack/ai/adapter-internals'
 import { defineSandbox } from '../src/sandbox'
 import { withSandbox } from '../src/middleware'
+import { captureLogger } from './fakes'
 import type { SandboxFileEvent } from '@tanstack/ai'
```

ESLint

[error] '`@tanstack/ai`' imported multiple times.

(import/no-duplicates)